x/profiles/types/msgs/msgs.go

```diff
@@ -152,61 +152,153 @@ func (msg MsgRequestDTagTransfer) GetSignBytes() []byte {
 func (msg MsgRequestDTagTransfer) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{msg.ReceivingUser}
+	return []sdk.AccAddress{msg.Sender}
 }
 
 // ----------------------
-// --- MsgAcceptDTagTransfer
+// --- MsgAcceptDTagTransferRequest
 // ----------------------
 
-// MsgAcceptDTagTransfer represent a DTag transfer acceptance message
-type MsgAcceptDTagTransfer struct {
-	NewDTag       string         `json:"new_d_tag" yaml:"new_dtag"`
-	CurrentOwner  sdk.AccAddress `json:"owner" yaml:"owner"`
-	ReceivingUser sdk.AccAddress `json:"receiving_user" yaml:"receiving_user"`
+// MsgAcceptDTagTransferRequest represent a DTag transfer acceptance message
+type MsgAcceptDTagTransferRequest struct {
+	NewDTag  string         `json:"new_d_tag" yaml:"new_dtag"`
+	Receiver sdk.AccAddress `json:"receiver" yaml:"receiver"`
+	Sender   sdk.AccAddress `json:"sender" yaml:"sender"`
 }
 
-// NewMsgAcceptDTagTransfer is a constructor for MsgAcceptDTagTransfer
-func NewMsgAcceptDTagTransfer(newDTag string, owner, receivingUser sdk.AccAddress) MsgAcceptDTagTransfer {
-	return MsgAcceptDTagTransfer{
-		NewDTag:       newDTag,
-		CurrentOwner:  owner,
-		ReceivingUser: receivingUser,
+// NewMsgAcceptDTagTransfer is a constructor for MsgAcceptDTagTransferRequest
+func NewMsgAcceptDTagTransfer(newDTag string, receiver, sender sdk.AccAddress) MsgAcceptDTagTransferRequest {
+	return MsgAcceptDTagTransferRequest{
+		NewDTag:  newDTag,
+		Receiver: receiver,
+		Sender:   sender,
 	}
 }
 
 // Route should return the name of the module
-func (msg MsgAcceptDTagTransfer) Route() string { return models.RouterKey }
+func (msg MsgAcceptDTagTransferRequest) Route() string { return models.RouterKey }
 
 // Type should return the action
-func (msg MsgAcceptDTagTransfer) Type() string { return models.ActionAcceptDtagTransfer }
+func (msg MsgAcceptDTagTransferRequest) Type() string { return models.ActionAcceptDtagTransfer }
 
-func (msg MsgAcceptDTagTransfer) ValidateBasic() error {
+func (msg MsgAcceptDTagTransferRequest) ValidateBasic() error {
 	if len(strings.TrimSpace(msg.NewDTag)) == 0 {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "new dTag can't be empty")
 	}
 
-	if msg.CurrentOwner.Empty() {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, fmt.Sprintf("invalid current owner address: %s", msg.CurrentOwner))
+	if msg.Receiver.Empty() {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, fmt.Sprintf("invalid current owner address: %s", msg.Receiver))
 	}
 
-	if msg.ReceivingUser.Empty() {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, fmt.Sprintf("invalid receiving user address: %s", msg.ReceivingUser))
+	if msg.Sender.Empty() {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, fmt.Sprintf("invalid receiving user address: %s", msg.Sender))
 	}
 
-	if msg.ReceivingUser.Equals(msg.CurrentOwner) {
+	if msg.Sender.Equals(msg.Receiver) {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "the receiving user and current owner must be different")
```

Comment:
I think we should change this to something like 

```suggestion
		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "the sender and receiver must be different")
```